pom.xml

```
@@ -8,7 +8,7 @@
     <parent>
         <groupId>org.carlspring.strongbox</groupId>
         <artifactId>strongbox-parent</artifactId>
-        <version>2.0-SNAPSHOT</version>
+        <version>1.0-ISSUE-1878-SNAPSHOT</version>
```

@tanvimehta could you please change this to 
```suggestion
        <version>1.0-issue-1878-dep-upgrade-SNAPSHOT</version>
```
on all files? It looks like there was some miscommunication - apologies for this!